covid19-etl/etl/ssr.py

```diff
@@ -46,16 +60,40 @@ def files_to_submissions(self):
         """
         Reads input files and converts the data to Sheepdog records
         """
-        self.parse_input_file()
+        print("Parsing file: {}".format(self.file_path))
+        extension = self.file_path.lower().split(".")[-1]
+        if extension == "txt":
+            self.parse_txt_input_file()
+        elif extension == "xlsx":
+            self.parse_xlsx_input_file()
+        else:
+            raise Exception(
+                f"I don't know how to parse extension {extension} for file {self.file_path}"
```

Comment:
Is there supposed to be an "f" here?

Reply:
yes, that's python 3 string formatting 😄  https://realpython.com/python-f-strings/#f-strings-a-new-and-improved-way-to-format-strings-in-python